code/modules/mob/living/carbon/carbon.dm

```diff
@@ -1233,13 +1232,13 @@ so that different stomachs can handle things in different ways VB*/
 	var/reagent_application = REAGENT_INGEST
 	var/requires_mouth = TRUE
 	var/instant = FALSE
-	var/efficiency = 1
+	var/how_many_reagents = medicine.reagents.total_volume
 
 	if(ispatch(medicine))
 		apply_method = "apply"
 		reagent_application = REAGENT_TOUCH
 		requires_mouth = FALSE
-		efficiency = 0.5 // Patches aren't that good at transporting reagents into the bloodstream
+		how_many_reagents = clamp(medicine.reagents.total_volume, 0.1, 2) // Patches aren't that good at transporting reagents into the bloodstream
```

Comment:
This looks like a behavior change / tweak. Why this efficiency change?

Reply:
Forgot to update description, this is how patches used to work, and I accidentally uncapped their reagent inject amounts.
I did forget to also do this in xeno code, applied!